src/Build/ElementLocation/ElementLocation.cs

```diff
@@ -85,29 +93,17 @@ public static ElementLocation EmptyLocation
             get { return s_emptyElementLocation; }
         }
 
-        /// <summary>
-        /// Get reasonable hash code.
-        /// </summary>
+        /// <inheritdoc />
         public override int GetHashCode()
         {
             // Line and column are good enough
-            return Line.GetHashCode() ^ Column.GetHashCode();
+            return Line ^ Column;
```

Comment:
This might lead to collisions with `Column ^ Line`

Reply:
Yeah it's no worse than before but seems pretty bad, can we at least get a bit-shift in there?

Reply:
Is this ever called? That would inform how much it matters

Reply:
I'll change it to `return Line ^ (Column * 397);` which matches some combination logic elsewhere in the repo.